README.md

```diff
@@ -6,99 +6,31 @@ A Cisco Webex Teams bot is an automated user within the Webex Teams platform tha
-
-To install Homebrew, open Terminal and run the following command.
-
-````sh
-ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
-````
-Once the installation is finished, install Git by running the following command.
-
-````sh
-brew install git
-````
-
-* **Install git on Ubuntu/Linux**
-
-To install Git on your machine, run each of the following commands in your terminal sequentially.
-
-````sh
-sudo apt-get update
-sudo apt-get upgrade
-sudo apt-get install git
-````
-
-* **Install git on Windows**
-
-Go to the [Downloading Git for Windows](https://git-scm.com/download/win) page, download and install it.  Keep all the settings standard as you run through the installation process.
-
-### Install Docker
-
-Docker is an open platform for developing, shipping, and running applications. It is the most trendy container technology for running modern software apps particularly in the cloud. Find out more about Docker and containers [here](https://www.docker.com/why-docker).
-
-The docker installation will depend on your operating system. After installing, confirm that Docker is available by entering the following command on terminal.
-
-````sh
-docker -v
-````
-
-* **Install Docker on MacOS**
-
-Go to the [Install Docker Desktop for Mac](https://docs.docker.com/docker-for-mac/install/) webpage. Head to Docker Hub by clicking the **Download the Docker Hub** button. After checking if your Apple Mac OS version fits the requirements, click on the **Get Docker** button and follow the installation instructions.
-
-* **Install Docker on Ubuntu/Linux**
-
-To set up the Docker repository in your machine, run each of the following command lines in your terminal sequentially.
-
-````sh
-sudo apt-get update
-sudo apt-get install apt-transport-https ca-certificates curl gnupg2 software-properties-common
-curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
-sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
-````
-
-Finally, to install Docker simply run
-
-````sh
-sudo apt-get update
-sudo apt-get install docker-ce
-````
-
-* **Install Docker on Windows**
-
-Go to the [Install Docker Toolbox on Windows](https://docs.docker.com/toolbox/toolbox_install_windows/) webpage and click on **Get Docker Toolbox for Windows** to start download. Follow the installation, accept admin prompts and requests to install network adapters.
-
-* **Install Docker on other OS**
-
-Please find instructions on how to install Docker on other Operating Systems [here](https://docs.docker.com/install/).
+"Ngrok exposes local servers behind NATs and firewalls to the public internet over secure tunnels." [Link to install ngrok](https://ngrok.com/download).
```

Comment:
I think calling it downloading ngrok rather than installing it might be clearer, because there's the step of unzipping it and copying it into the bot folder

Reply:
agreed :)